database/migrations/sql/20180604_06_create_users.sql

```diff
@@ -32,11 +33,10 @@ CREATE TABLE access_role (
 
 
 CREATE TABLE permission (
-  permission_id SERIAL NOT NULL UNIQUE,
-  entity        VARCHAR NOT NULL UNIQUE,
-  read_access   ENUM_permission_type NOT NULL,
-  write_access  ENUM_permission_type NOT NULL,
-  delete_access ENUM_permission_type NOT NULL,
+  permission_id      SERIAL NOT NULL UNIQUE,
+  entity             VARCHAR NOT NULL UNIQUE,
+  permission_type    ENUM_permission_type NOT NULL,
+  permission_access  ENUM_permission_access NOT NULL,
```

Comment:
So is `entity` here going to be `organisation_details`, `constants`, etc?

Reply:
Yup. Should I make this an enum for now, as these shouldn't change in the future?

Reply:
Nah it's fine like this I think